src/pages/Home.tsx

```diff
@@ -1,5 +1,5 @@
 import { IGitHubResponse, useFetch } from "../components/useFetch";
-import logo from "../nina_logo.png";
+import logo from "../logo-BLACK.webp";
```

Comment:
change the name. Use this naming convention for assets
`nina_type_color`

in this case:
`nina_logo_black`